content/en/tracing/advanced/connect_logs_and_traces.md

```diff
@@ -30,9 +30,9 @@ Your language level logs *must* be turned into Datadog attributes in order for t
 {{< tabs >}}
 {{% tab "Java" %}}
 
-Enable injection in the Java Tracer's [configuration][1] by setting `Ddd.logs.injection=true` or through environment variable `DD_LOGS_INJECTION=true`.
+Enable injection in the Java Tracer's [configuration][1] by adding `-Ddd.logs.injection=true` as a jvm startup argument or through environment variable `DD_LOGS_INJECTION=true`.
```

Comment:
```suggestion
Enable injection in the Java tracer's [configuration][1] by adding `-Ddd.logs.injection=true` as a jvm startup argument or through environment variable `DD_LOGS_INJECTION=true`.
```

Reply:
Should there be a capital `D` in `Ddd.logs.injection`?

Reply:
It should either be `-Ddd.logs.injection=true` which identifies the whole setting, or just `dd.logs.injection=true`.  the `-D` is part of the java command line flag signifying a system property.